exams/management/commands/authorization_user_exam.py

```diff
@@ -32,41 +28,29 @@ def add_arguments(self, parser):
             type=int
         )
         parser.add_argument(
-            "--course-key",
-            dest="course_key",
-            default=None,
-            help="Edx course key where we want to authorize users who passed course(s).",
+            "--all",
+            dest="all",
+            default=True,
+            help="Authorize eligible users for exams",
             type=int
         )
 
     def handle(self, *args, **options):
         """
         Trigger exam authorizations for user for a course if he has passed and paid for it.
         """
-        # pylint: disable=too-many-locals
+        all = options.get("all", False)
         program_id = options.get("program_id", None)
         username = options.get("username", None)
-        course_key = options.get("course_key", None)
-
-        programs = Program.objects.filter(exam_series_code__isnull=False, live=True).filter(~Q(exam_series_code=""))
-        if program_id:
-            programs = programs.filter(id=program_id)
-
-        for program in programs:
-            users_qset = ProgramEnrollment.objects.filter(program=program)
-            if username:
-                users_qset = users_qset.filter(user__username=username)
-
-            users = users_qset.values_list('user', flat=True)
-
-            for user in users:
-                enrollments_qset = CachedEnrollment.user_course_qset(user, program=program)
-                if course_key:
-                    enrollments_qset = enrollments_qset.filter(course_run__edx_course_key=course_key)
-
-                mmtrack = get_mmtrack(user, program)
 
-                for enrollment in enrollments_qset:
-                    # if user has passed and paid for the course
-                    # and not already authorized for exam the create authorizations.
-                    authorize_for_exam(mmtrack, enrollment.course_run)
+        if all:
+            bulk_authorize_for_exam()
+            self.stdout.write(self.style.SUCCESS('Done with authorization process'))
+        elif program_id or username:
+            bulk_authorize_for_exam(
+                program_id=program_id,
+                username=username
+            )
+            self.stdout.write(self.style.SUCCESS('Done with authorization process'))
+        else:
+            self.stdout.write(self.style.SUCCESS('No param is provided.'))
```

Comment:
This should be modified to not return a success message (there is no success here): this should return the `--help` option.